lms/static/js/student_account/views/AccessView.js

```diff
@@ -85,9 +86,14 @@
             },
 
             render: function() {
-                $(this.el).html( _.template(this.tpl)({
+                var output = HtmlUtils.template(this.tpl)({
                     mode: this.activeForm
-                }));
+                });
+
+                HtmlUtils.setHtml(
```

Comment:
The good news is that ``HtmlUtils.template`` returns an HTML snippet, so you can just do:

    HtmlUtils.setHtml(this.$el, output);